### ISSUE TITLE
Wrong Account Entirely? 

### ISSUE THREAD

**Issue by Azrielexen** [NECROBOTIO/NecroBot#1682]
I've been using the same account since the original version. For some reason instead of saying playing as Azriel****(last stars the last part of my username) it says playing as my exact email address and is using an entirely new account that started at level 0. On my phone I can still login using the same username and password as Azriel**** like I said, the bot appears to have given me a whole new account?

**Comment by mmelzian** [comment]
I am having the same issue.
With the old version (get the token via the google website) I was able to choose if I want to use the account with the email address, or the alias. I think this was done in YouTube when they started to use real names. There I had the option to create another user with a nick name.
So both accounts have the same email address...

Interestingly these 2 accounts seems to be separated at google (or PoGo) so with this version now (0.5) I cannot choose which account should be taken so I started at level 0 again.

I tried to fix it in the code to find the place where this happened but I don't get how the login works...
